.copy-edit-stylesheet-checklist.md

```diff
@@ -0,0 +1,50 @@
+Basic checklist/stylesheet used for copy editing BOLTs —
+temporarily included here for reference purposes.
+
+  - spelling
+  - typos
+  - sentence structure
+    - sentence fragments
+    - run-on sentences
+    - dangling, misplaced modifiers
+    - consistent paragraph tense (e.g. past, present, future)
+    - passive voice (e.g. avoid 'we')
+      - exception: `Introduction` section
+  - capitalization
+    - table of contents
+    - headers
+    - commonly forgotten: 'Bitcoin', 'Lightning', 'ID'
+  - punctuation
+    - correct comma, colon, semi-colon, em-dash placement
+    - appropriate use of parenthesis
+  - formatting
+    - single spaces between sentences
+    - consistent use of _emphasis_, **strong**, `code`, CAPS, 'quotes'
+    - single line separators between paragraphs and page elements
+    - type numbers (e.g. '1', '1s') rather than write them (e.g. 'one', 'ones')
+      - exception: 'non-zero'
```

Comment:
Yes, exactly right, generally small numbers are written out, however, it's largely a stylistic choice depending upon the publication. My thought was due to the many instances of written mathematical statements, this would help to avoid any potential confusion (e.g. "the old one must be equal to 1"). But this does look funny in a non-mathematical sentence. 

So how about this: type references to actual digits (e.g. "the number 1"), otherwise write out small amounts (e.g. "there is one instance of the number 2", "there are two number 1s").